src/index.html

```diff
@@ -7,8 +7,100 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link 
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        class="homebutton"
+        href="#home"
+        alt="logo and link to homepage"
+      >
+      </a>
+      <nav class="nav">
+        <ul class="menu">
+          <li class="menu__item menu__item-left">
+            <a href="#apple" class="menu__item_link is-active">Apple</a>
+          </li> 
+          <li class="menu__item">
+            <a href="#samsung" class="menu__item_link">Samsung</a>
+          </li>
+          <li class="menu__item">
+            <a href="#smartphones" class="menu__item_link">Smartphones</a>
+          </li>
+          <li class="menu__item">
+            <a 
+              href="#laptopsandcomputers"
+              data-qa="hover"
+              class="menu__item_link"
+             >Laptops & Computers
+            </a>
+          </li>
+          <li class="menu__item">
+            <a href="#gadgets" class="menu__item_link">Gadgets</a>
+          </li>
+          <li class="menu__item">
+            <a href="#tablets" class="menu__item_link">Tablets</a>
+          </li>
+          <li class="menu__item">
+            <a href="#photo" class="menu__item_link">Photo</a>
+          </li>
+          <li class="menu__item menu__item-right">
+            <a href="#video" class="menu__item_link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <header class="header header-2">
+      <a
+        class="homebutton"
+        href="#home"
+        alt="logo and link to homepage"
+      >
+      </a>
+      <nav class="nav">
+        <ul class="menu menu-2">
+          <li class="menu__item menu__item-left">
+            <a href="#apple" class="menu__item_link is-active">Apple</a>
+          </li> 
+          <li class="menu__item">
+            <a href="#samsung" class="menu__item_link">Samsung</a>
+          </li>
```

Comment:
Put empty line between open & close tag
```suggestion
          <li class="menu__item menu__item-left">
            <a href="#apple" class="menu__item_link is-active">Apple</a>
          </li> 
          
          <li class="menu__item">
            <a href="#samsung" class="menu__item_link">Samsung</a>
          </li>
```